anndata/core/sparsedataset/__init__.py

```diff
@@ -60,21 +60,21 @@ class backed_csc_matrix(BackedSparseMatrixMixin, ss.csc_matrix):
 
 
 def get_format_str(data):
-    for fmt, backed_class, memory_class in FORMATS:
+    for fmt, _, memory_class in FORMATS:  # To appease codacy
```

Comment:
This is just regular pythonic code, nothing about codacy :wink:

Reply:
I just don't think it's worth a  ❌.

On a related note, what's up with test coverage?